packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -324,6 +347,22 @@ const CommentBoard = ( { thread, onResolve, onEdit, onDelete, status } ) => {
 					) }
 				</VStack>
 			</HStack>
+			{ 'reopen' === actionState && (
```

Comment:
I would suggest removing both this and the resolving confirmation dialog. I'm curious how others feel about this.

Reply:
Since these are non-destructive actions, I think removing the friction would be good.